src/libraries/Microsoft.Extensions.Caching.Memory/ref/Microsoft.Extensions.Caching.Memory.cs

```diff
@@ -4,6 +4,9 @@
 // Changes to this file must follow the https://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+using System;
+using System.Runtime.CompilerServices;
```

Comment:
Generally the `ref` source does not have `using`s. Every type is qualified with its own namespace.

Reply:
474041ddfce80ea9eb98c2937a618f70720b2e9f